doc/release-notes/9276-doc-cvoc-index-in.md

```diff
@@ -0,0 +1,8 @@
+## Release Highlights
+
+### Updates on Support for External Vocabulary Services
+
+#### Indexed field accuracy
```

Comment:
Suggested change: 
### Improved Indexing with Compound Fields

When using an external vocabulary service with compound fields, you can now specify which field(s) will include additional indexed information, such as translations of an entry into other languages. This is done by adding the  `indexIn` in `retrieval-filtering`.
For more information, please check [GDCC/dataverse-external-vocab-support documentation](https://github.com/gdcc/dataverse-external-vocab-support/tree/main/docs).